README.md

```diff
@@ -1,13 +1,90 @@
+- Use a provided class by creating object instances using the new keyword
+- Manipulate the page after it has loaded adding, removing, and updating elements on the DOM
+- Begin to understand the connection between HTML, CSS and JavaScript
 
-Project spec & rubric can be found [here](https://frontend.turing.io/projects/module-1/hang-in-there.html)
+#### Learning Goals
+- Develop a better understanding of learned concepts
+- Work on teaching strategies to grow as a developer
+- Get more reps and add to toolbox and tying concepts together
+- Work on teamwork and remote collaboration skills
+
+#### Why was this project created?
+- This project was created to teach the fundamentals of working on a team and implementing vanilla JavaScript to create a basic web application
+
+#### How does this work?
+![homepage screenshot](link in the parens)
+- Each page load will automatically generate a random motivational poster. From here, the user has options!
+- The user may continue to generate random posters by clicking the `Show Another Random Poster Button`.
+- To create their own poster, a user will click on `Make Your Own Poster`. This will pull up a form that accepts inputs for an image URL, a title, and a quote. Clicking `Show my poster` will return the user to the home page and display their new poster! There is also an option to return to the home page without creating a poster, by clicking `Nevermind, take me back!`
+- Posters can also be stored for later viewing by clicking `Save This Poster`. This switches to a grid view displaying mini versions of all saved posters.
+![saved posters view + delete gif](link in the parens)
+- To delete a poster, the user can double click the image.
+- This applications runs directly in the web browser!
+
+#### Want to contribute?
+- Interested in contributing to Hang in there?!
+- Fork this project and clone to your machine.
+- Create a pull request, and one of our project managers will get back to you with a code review!
+
+#### Technologies Used
+- JavaScript
+- HTML
+- CSS
+
+#### Project Reflection
+##### Nichele
+- I had a great experience during this project. Peter and I spent a good deal of time planning and discussing the problems, challenges we might face, and how to address problems when encountered.
+##### Peter
+
+##### Wins
+##### Nichele
+- I felt like our communication styles worked well together, and we were able to address coding challenges through discussion and solo pseudocoding.
+- Planning! We spent a lot of time in discussion deciding how to address a problem before writing any code. This seemed to give us a clear direction without leaving us muddled in issues we hadn't accounted for.
+- Our git workflow seemed to come together pretty quickly and I didn't have any issues with merges or pulls.
+- We were able to reach out to mentors when we needed assistance, and were able to come up with a solution to our problem with some gentle nudging.
+##### Peter
+
```

Comment:
- We struck an effective balance between cooperative and solo work, always agreeing on when it was best to continue discussion and/or work cooperatively on a given feature/fix, and when to break away and analyze the issue independently, regrouping afterwards.
- I think we demonstrated self-awareness of our own abilities, with a willingness to push through points of frustration in order to improve our debugging skills, but not to the point that we ever struggled unproductively.  We knew when to reach out to our peers/mentors, and then using their guidance were able to continue on and solve these problems ourselves.

Reply:
√